package.json

```diff
@@ -1,10 +1,12 @@
 {
-  "name": "logdna-winston",
+  "name": "@logdna/winston",
```

Comment:
well, that's pretty confusing, I think - renaming the `npm` package name basically means creating a new package but it starts from `2.3.2` - I'd suggest, @darinspivey, not to add namespace for now or if we add, we should deprecate this repo and start the new one named `logger-winston` and start the versioning from `1.0.0` to keep the consistency

Reply:
reverted.  I don't think we need to spend time worrying about the naming for this.  Let's move on and just keep it as-is.